src/main/java/org/assertj/core/error/ShouldContainCharSequence.java

```diff
@@ -125,6 +125,60 @@ public static ErrorMessageFactory shouldContainIgnoringCase(CharSequence actual,
                                          StandardComparisonStrategy.instance());
   }
 
+  /**
+   * Creates a new <code>{@link ShouldContainCharSequence}</code>.
+   *
+   * @param actual   the actual value in the failed assertion.
+   * @param sequence the sequence of values expected to be in {@code actual}.
+   * @return the created {@code ErrorMessageFactory}.
+   */
+  public static ErrorMessageFactory shouldContainIgnoringWhitespace(CharSequence actual, CharSequence sequence) {
```

Comment:
rename `sequence` to `value`, here it's not a sequence of values

Reply:
You're sure about this. All other already exising methods in this class use `CharSequence sequence` as parameter.